models/src/test/scala/coop/rchain/models/blockImplicits.scala

```diff
@@ -38,12 +38,11 @@ object blockImplicits {
 
   implicit val arbitraryJustification: Arbitrary[Justification] = Arbitrary(justificationGen)
 
-  val processedDeployGen: Gen[ProcessedDeploy] =
+  val signedDeployDataGen: Gen[Signed[DeployData]] =
     for {
-      term        <- listOfN(32, Gen.alphaNumChar).map(_.mkString)
-      timestamp   <- arbitrary[Long]
-      (sec, pub)  = Secp256k1.newKeyPair
-      bytesLength <- Gen.chooseNum(128, 256)
+      term      <- listOfN(32, Gen.alphaNumChar).map(_.mkString)
```

Comment:
Is there a reason not to use `arbitrary[String]`?

Reply:
It will generate non-alphanum characters which I believe causes some problems. The length though could be arbitrary choosen instead. I will fix it.